packages/protocol/contracts/governance/Governance.sol

```diff
@@ -98,11 +53,19 @@ contract Governance is
   uint256 public minDeposit;
   mapping(address => uint256) public refundedDeposits;
   mapping(address => ContractConstitution) private constitution;
-  mapping(uint256 => Proposal) private proposals;
+  mapping(uint256 => Proposals.Proposal) private proposals;
   mapping(address => Voter) public voters;
   SortedLinkedList.List private queue;
   uint256[] public dequeued;
   uint256[] public emptyIndices;
+  // The average network participation in governance, weighted toward recent proposals.
+  int256 public participationBaseline;
+  // The lower bound on the participation baseline.
+  int256 public participationFloor;
+  // The weight of the most recent proposal's participation on the baseline.
+  int256 public participationUpdateCoefficient;
+  // The fraction of the baseline under which the proposal will be padded with "no" votes.
+  int256 public criticalBaselineLevel;
```

Comment:
Do we need this to be adjustable?

Reply:
This is governable and fixed, it represents a fraction of the moving baseline where quorum kicks in. In Tezos, they use 80% (iirc) of the baseline to do quorum. We use 100% as default to be conservative, with the option to change it.